addons/website/static/src/js/content/snippets.animation.js

```diff
@@ -1329,8 +1334,22 @@ registry.WebsiteAnimate = publicWidget.Widget.extend({
                 this.$target.find('.modal:visible').scrollTop() :
                 this.$scrollingElement.scrollTop();
             const elTop = this._getElementOffsetTop($el[0]) - scrollTop;
-
-            const visible = this.windowsHeight > (elTop + elOffset) && 0 < (elTop + elHeight - elOffset);
+            let visible;
+            const footerEl = el.closest('.o_footer_slideout');
```

Comment:
Shouldn't you also check if there is indeed `o_footer_effect_enable` on the `#wrapwrap` so as not to pass in the code for `o_footer_slideout` when it is not needed.